### PR TITLE
Migrate docs

### PR DESCRIPTION
**Pull request by aszecowka**
Get rid of documentation for Azure Broker and GCP Broker and move them to the bundles.

How to test it:
- run kyma from this PR
- use this PR https://github.com/kyma-project/bundles/pull/82 and script `bundles/scripts/deploy_current_bundles_to_minikube.sh`

Be aware that there is a reported issue: https://github.com/kyma-project/kyma/issues/3907

### CODE REVIEW

File: docs/helm-broker/docs/03-04-bundles-docs.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Using the Helm Broker, you can provision a bundle which provides its own Service
 
 ```
 apiVersion: cms.kyma-project.io/v1alpha1
-kind: ClusterDocsTopic
```
Comment (mszostok) [inline docs/helm-broker/docs/03-04-bundles-docs.md]: user can define both the ClusterDocsTopic and DocsTopic, so you can have two kinds in one examples. Please also fix the documentation about it. Right now we are mentioning only the ClusterDocsTopic. 

Maybe we can also mention that the name of the file and place where it is stored is up to the user. It does not have to be called `docs.yaml`

File: docs/helm-broker/docs/03-04-bundles-docs.md [hunk 2]
```diff
@@ -19,21 +20,22 @@ spec:
...
       filter: docs/{class_name}/ 
 ___
 apiVersion: cms.kyma-project.io/v1alpha1
-kind: ClusterDocsTopic
+kind: DocsTopic
 metadata ... 
 ```
-For detailed descriptions of all parameters, see the [ClusterDocsTopic custom resource](/components/headless-cms/#custom-resource-clusterdocstopic). 
-For more information about currently supported types of the assets, read [this](/components/headless-cms/#overview-overview-headless-cms-in-kyma) document.
+If your bundle defines `ServiceBroker`, use `DocsTopic` type. If your bundle defines `ClusterServiceBroker`, use `ClusterDocsTopic` type.
```
Comment (klaudiagrz) [inline docs/helm-broker/docs/03-04-bundles-docs.md]:
```suggestion
If your bundle defines a ServiceBroker, use the DocsTopic type. If your bundle defines a ClusterServiceBroker, use the ClusterDocsTopic type.
```

File: docs/helm-broker/docs/03-04-bundles-docs.md [hunk 3]
```diff
@@ -19,21 +20,22 @@ spec:
...
 
+For detailed descriptions of all parameters, see the [ClusterDocsTopic custom resource](/components/headless-cms/#custom-resource-clusterdocstopic)
```
Comment (klaudiagrz) [inline docs/helm-broker/docs/03-04-bundles-docs.md]:
```suggestion
For more details, see the [ClusterDocsTopic](/components/headless-cms/#custom-resource-clusterdocstopic)
```

File: docs/helm-broker/docs/03-04-bundles-docs.md [hunk 4]
```diff
@@ -19,21 +20,22 @@ spec:
...
+or [DocsTopic custom resource](/components/headless-cms/#custom-resource-docstopic).
```
Comment (klaudiagrz) [inline docs/helm-broker/docs/03-04-bundles-docs.md]:
```suggestion
and [DocsTopic](/components/headless-cms/#custom-resource-docstopic) custom resources documentation.
```

File: docs/helm-broker/docs/03-04-bundles-docs.md [hunk 5]
```diff
@@ -19,21 +20,22 @@ spec:
...
+For more information about currently supported types of the assets, read [this](/components/headless-cms/#overview-overview-headless-cms-in-kyma) document.
 
-One ClusterDocsTopic object corresponds to a single Service Class with the same ID as the name of the specified ClusterDocsTopic.
-Place all ClusterDocsTopic definitions for your bundle in the `docs.yaml` file.
-Store documentation for each Service Class in the `docs/{class_name}` directory which corresponds to the value of the **filter** parameter in the ClusterDocsTopic definition.
 
+One ClusterDocsTopic or DocsTopic object corresponds to a single Service Class with the same ID as the name of the specified ClusterDocsTopic or DocsTopic.
```
Comment (klaudiagrz) [inline docs/helm-broker/docs/03-04-bundles-docs.md]:
```suggestion
One ClusterDocsTopic or DocsTopic object corresponds to a single Service Class with the same ID as the name of the specified object.
```

**Pull request merged**